capstone/src/main/webapp/script.js

```diff
@@ -13,16 +13,44 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Call the perspective API
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+async function callPerspective(text, lang) {
+  const response = await fetch('/call_perspective', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
```

Comment:
Could we combine this all into one line?

Reply:
Will do!